Project-1/README.md

```diff
@@ -1,5 +1,7 @@
 # Project 1 Assignment 
 
+(test)
+
```

Comment:
Thanks! I've created a new pull request to address your suggestions.

For question 3, I think we should also add the non-identity statements for the other definitions, not just 3(a). For 3(b) I added x≠y∧x≠z∧x≠w∧w≠y∧w≠z∧y≠z, for 3(c) I added x≠v∧x≠y∧x≠z∧x≠w∧w≠y∧w≠z∧w≠v∧y≠z∧y≠v∧v≠z, for 3(d) I added y≠z∧y≠x, and for 3(e) I added v≠w∧v≠x∧v≠y∧v≠z.

For question 4, thank you for pointing out inverse roles. I didn't know about them, which left me totally lost about how to answer this. Should the ≥2 be written before or after ∃parent_of? The book makes it look like it comes before.

Thanks for the recommendation about mermaids, and your solution to 9(c) looks good to me!

Reply:
> Thanks! I've created a new pull request to address your suggestions.
> 
> For question 3, I think we should also add the non-identity statements for the other definitions, not just 3(a). For 3(b) I added x≠y∧x≠z∧x≠w∧w≠y∧w≠z∧y≠z, for 3(c) I added x≠v∧x≠y∧x≠z∧x≠w∧w≠y∧w≠z∧w≠v∧y≠z∧y≠v∧v≠z, for 3(d) I added y≠z∧y≠x, and for 3(e) I added v≠w∧v≠x∧v≠y∧v≠z.
> 
> For question 4, thank you for pointing out inverse roles. I didn't know about them, which left me totally lost about how to answer this. Should the ≥2 be written before or after ∃parent_of? The book makes it look like it comes before.
> 
> Thanks for the recommendation about mermaids, and your solution to 9(c) looks good to me!

@Finn1928 You know, I am a starter of description logic. When I search for some materials of DL, I find it is both fine that ≥2 is written before or after.

For Q3, you are definitely right and careful about non-identity statements, though I'm a bit hesitant to do that because that makes our formulas pretty ugly 😢

Reply:
"I'm a bit hesitant to do that because that makes our formulas pretty ugly" True!